<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QWidget" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>517</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="AppLayout">
    <item>
     <layout class="QVBoxLayout" name="outputLayout">
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <widget class="QLineEdit" name="errorLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="inputLayout">
      <item>
       <layout class="QVBoxLayout" name="totalByCompanyLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Company Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="companyLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="totalOutputLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Total</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="totalResultlineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="calculateTotalButton">
            <property name="text">
             <string>Calculate Total</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="filterLayout">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Paid/Unpaid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filterButton">
          <property name="text">
           <string>Filter by isPaid</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="addLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Company Name</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="addCompanyNameLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Serial Number</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="addSerialNumberLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Sum</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="addSumLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="addIsPaidCheckBox">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Is Paid</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="showAllButton">
        <property name="text">
         <string>Show All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
